be/src/cache/object_cache/object_cache.h

```diff
@@ -32,9 +36,14 @@ class ObjectCache {
     // Init the object cache instance with options.
     Status init(const ObjectCacheOptions& options);
 
+#ifdef WITH_STARCACHE
+    // Init the object cache instance with an exist block cache module.
+    Status init(const std::shared_ptr<starcache::StarCache>& star_cache);
```

Comment:
if the `ObjectCache` is designed to encapsulate the underlying blockcache or starcache, shouldn't this interface be hidden behind the scenes?

Reply:
I will fix it later pr